misc/package/Data Files/MWSE/core/mcm/components/categories/Category.lua

```diff
@@ -66,24 +66,22 @@ function Category:update()
 end
 
 function Category:checkDisabled()
+	-- dont disable if there are no subcomponents
+	if next(self.components) == nil then return false end
```

Comment:
Is this condition ever reached? Considering we had `for _, component in ipairs(self.components) do` below without it ever erroring (at least to my knowledge), it looks unnecessary.

Edit: I see `checkDisabled` is called under a bit different conditions now, and this check makes sense if we encounter an empty Category. It's ok then, but let's use our standard library's `table.empty`.